src/Client.js

```diff
@@ -235,23 +255,25 @@ export default class Client extends EventTarget {
 
   /**
    * Create a new alias with the passed in aliasName and payload.
-   * @param {string} aliasName the name of the new alias
-   * @param {string} payload the payload on the new alias.
+   * @param {Object} options
+   * @param {string} options.aliasName the name of the new alias
+   * @param {string} options.payload the payload on the new alias.
    * @returns {Promise<Alias>} The Alias model of the newly created alias.
    */
-  createAlias(aliasName, payload) {
+  createAlias({aliasName, payload} = {}) {
     return this.aliases.createAlias(aliasName, payload);
   }
 
   /**
    * Update an existing alias.
-   * @param {Alias} alias the model of the alias to update
-   * @param {string} [name] the optional new name for the alias
-   * @param {string} [payload] the optional new payload for the alias
+   * @param {string} aliasName the name of the alias to update.
+   * @param {Object} updates object containing the updates to the alias.
+   * @param {string} [updates.aliasName] the optional new name for the alias
+   * @param {string} [updates.payload] the optional new payload for the alias
    * @returns {Promise<Alias>} The Alias model of the newly updated alias.
    */
-  updateAlias(alias, name, payload) {
-    return this.aliases.updateAlias(alias, name, payload);
+  updateAlias(aliasName, { aliasName: newName, payload: newPayload } = {}) {
```

Comment:
not sure if we want to default to the current account username for the update method.
Feels like you actually want to specify the alias since you're updating the alias.